skbase/lookup/_lookup.py

```diff
@@ -257,42 +276,44 @@ def _walk(root, exclude=None, prefix=""):
 
 
 def _import_module(
-    module_name: str,
+    module: Union[str, importlib.machinery.SourceFileLoader],
     suppress_import_stdout: bool = True,
-    loader: importlib.machinery.SourceFileLoader = None,
 ) -> ModuleType:
     """Import a module, while optionally suppressing import standard out.
 
     Parameters
     ----------
-    module_name : str
-        Name of the module to be imported.
+    module : str or importlib.machinery.SourceFileLoader
```

Comment:
were the old `module` and `loader` args always mutually exclusive?

Reply:
looking through the old code, I suppose so - makes sense to unify the argument therefore

Reply:
Yea, it was redundant to have the two params. It was just one param, but doing different things based on type.